core/trino-server/src/main/provisio/trino.xml

```diff
@@ -25,6 +25,12 @@
         <artifact id="${project.groupId}:trino-server-main:${project.version}" />
     </artifactSet>
 
+    <!-- Configuration Resolver -->
+    <artifactSet to="configuration-resolver/trino-configuration-resolvers">
```

Comment:
Should we use the plugin directory or have a new directory for the configuration-resolvers ?

Reply:
This should use a new directory, as these are not standard Trino plugins. It is a new plugin type.